EXAMPLES/PYARPACK/pyarpackSparseQR.py.in

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
```

Comment:
I guess the .in could be renamed as we don't plan to run with py2?

Reply:
As you like. I added `.in` as there are some `*.py.in` which can _not_ be directly used as python scripts (need replacement of `PYINT` variable to handle **I**LP64 case... But the repo does not compile in ILP64 : I discovered that _after_ which led me to others problems likely related to #230)